scripts/scripts.js

```diff
@@ -561,6 +561,26 @@ function buildAuthorBox(main) {
   main.append(div);
 }
 
+// Properly decorate fragments that were pulled in
+document.addEventListener('aem:experimentation', (ev) => {
+  // Do not redecorate the default content
+  if (ev.detail.variant === 'control' || ev.detail?.campaign === 'default' || ev.detail?.audience === 'default') {
+    return;
+  }
+  // Rebuild the autoblock as needed
+  if (ev.detail.element.classList.contains('hero')) {
+    const parent = ev.detail.element.parentElement.parentElement;
+    [...ev.detail.element.children].reverse().forEach((el) => parent.prepend(el));
+    ev.detail.element.remove();
+    decorateBlocks(parent);
+    loadBlocks(parent);
+  } else if (ev.detail.element.classList.contains('block')) {
```

Comment:
You can probably remove this condition here and just keep the default block decoration below